### PR TITLE
PORTALS-2499, PORTALS-2502, PORTALS-2503

### PR DESCRIPTION
**Pull request by hallieswan**
Before: 

https://user-images.githubusercontent.com/26949006/232628884-f786ad22-98c8-4aee-9d89-1c511255015b.mov

After: 

https://user-images.githubusercontent.com/26949006/232628842-57df8f4f-de76-4656-ba0a-ecb984375ef7.mov

### CODE REVIEW

File: packages/synapse-react-client/src/lib/containers/access_requirements/HasAccessV2.tsx [hunk 1]
```diff
@@ -80,6 +80,8 @@ function AccessIcon(props: { downloadType: FileHandleDownloadTypeEnum }) {
         label={configuration.tooltipText}
         sx={{
           color: configuration.color(theme),
+          height: '16px',
+          verticalAlign: 'text-top',
```
Comment (hallieswan) [inline packages/synapse-react-client/src/lib/containers/access_requirements/HasAccessV2.tsx:84]: [PORTALS-2499](https://sagebionetworks.jira.com/browse/PORTALS-2499)

Before: 
main_LockIconOnly

After: 
feature_LockIconOnly

File: packages/synapse-react-client/src/lib/containers/table/ExpandableTableDataCell.tsx [hunk 2]
```diff
@@ -54,6 +54,8 @@ export default function ExpandableTableDataCell(
             icon={isExpanded ? 'minusBoxOutline' : 'addBoxOutline'}
             sx={{
               color: 'grey.600',
+              height: '16px',
+              verticalAlign: 'top',
```
Comment (hallieswan) [inline packages/synapse-react-client/src/lib/containers/table/ExpandableTableDataCell.tsx:58]: [PORTALS-2503](https://sagebionetworks.jira.com/browse/PORTALS-2503)

File: packages/synapse-react-client/src/lib/style/components/_access-requirement-list.scss [hunk 3]
```diff
@@ -69,6 +69,7 @@
     background-color: SRC.$primary-action-color;
     padding: 10px 70px 10px 70px;
     color: white;
+    border-radius: SRC.$button-border-radius;
```
Comment (hallieswan) [inline packages/synapse-react-client/src/lib/style/components/_access-requirement-list.scss:72]: Adjust border-radius for buttons in Access Requirements List, missed in [PORTALS-2597](https://sagebionetworks.jira.com/browse/PORTALS-2597)

Before: 
main_AccessList

After: 
feature_AccessList

File: packages/synapse-react-client/src/lib/style/base/_core.scss [hunk 4]
```diff
@@ -1199,7 +1199,7 @@ hr ~ .SRC-wrapper {
 }
 
 .table > tbody > tr > td {
-  vertical-align: middle;
+  vertical-align: top;
```
Comment (hallieswan) [inline packages/synapse-react-client/src/lib/style/base/_core.scss:1202]: [PORTALS-2502](https://sagebionetworks.jira.com/browse/PORTALS-2502?focusedCommentId=170374)

**Review by jay-hodgson — APPROVED** [review]

**Pull request merged**